src/lib/plot/hint.js

```diff
@@ -215,15 +269,18 @@ class Hint extends PureRenderComponent {
 
   render() {
     const {
+      className,
+      style,
       value,
       format,
       children} = this.props;
 
-    const {style, className} = this._getPositionInfo();
+    const posInfo = this._getPositionInfo();
     return (
       <div
-        className={`rv-hint ${className}`}
+        className={`rv-hint ${posInfo.className} ${className}`}
         style={{
+          ... posInfo.style,
           ... style,
```

Comment:
Will drop this line.  Related to comment above about adding className and style to Hint props; this merits a broader discussion for the react-vis library